src/components/Definitions/Definitions.scss

```diff
@@ -20,10 +20,25 @@ limitations under the License.
 }
 
 main {
-  .bx--structured-list {
-    background-color: white;
+  .bx--structured-list-th {
+    background-color:  #7190c1;
+    border: 2px solid white;
+    padding: 1rem;
+  }  
+
+
+  .bx--structured-list-row:nth-child(odd) {
```

Comment:
we should use the `useZebraStyles` prop on the DataTable component instead of adding our own styles here

Reply:
Most of the tables currently used in the UI are structured lists, not data tables, and I thought what we discussed involved leaving these as structured lists for the time being.. so I'm not sure this would be possible?

Reply:
@AlanGreene see the comments above

Reply:
Yes that's right, I didn't notice the `structured-list` portion of the class name here, just the odd/even rules and wrongly assumed this was targetting some of our DataTables. The structured lists should be left as they are for now, so these styles can be removed.